classifiers/python_functions/cosine_similarity/code_snippet.md

```diff
@@ -4,7 +4,7 @@ from numpy import dot
 from numpy.linalg import norm
 from sklearn.feature_extraction.text import TfidfVectorizer
 
-YOUR_ATTRIBUTE = "your-text"
+YOUR_ATTRIBUTE: str = "text"
```

Comment:
not 100% sure what the idea for this is. Shouldn't a classifier always return some kind of label? so is this maybe a generator instead??

Reply:
also is the text supposed to be static? also a variable i'd assume

Reply:
You are right, the text is supposed to be variable. 

I think the idea is that based on the similarity score, it decides whether the two statements are similar or not. As long as the modules returns the score, it can be a generator. But I think we can define some threshold(s) on the scores where it classifies the texts into **not-similar**, **borderline similar**, and **similar**.

Reply:
Constants have been renamed. Resolving.